### ISSUE TITLE
String sort is case sensitive

### ISSUE THREAD

**Issue by tero-kiituri** [pocketbase/pocketbase#4580]
First of all thanks @ganigeorgiev! Pocketbase is amazing.

I noticed that string sort is case sensitive. In the sorting order first comes all strings with upper case letters and only after that strings with lower case letters. E.q. AAA, ZZZ, aaa, zzz. This can be produced with admin ui and api.

I understand that this may be by design and/or something that comes from Sqlite.  However, I think this a bit confusing and could be considered being a bug. (At least my users consider this being a bug :) )

According to my experience human reader expects alphabetical ordering being case insensitive. Case insensitive ordering would produce e.q. AAA, aaa, ZZZ, zzz. Other as valid option is to put the lower case letters before upper case letters.

**Comment by ganigeorgiev** [comment]
When the `sort` parameter is used it is translated to plain `ORDER BY col ASC/DESC` statement with the default collation.

Option to specify custom collation (including natsort) is in the [roadmap](https://github.com/orgs/pocketbase/projects/2/views/1?filterQuery=collation&pane=issue&itemId=33936705) but remains a low priority.

**Comment by ghostdevv** [comment]
Is this still planned? I wasn't able to find it in the roadmap

**Comment by ganigeorgiev** [comment]
No, it was discarded because it ended up complicating a lot of things around the custom SQLite drivers support.

Depending on your use case, a workaround could be to try with the [`:lower` modifier](https://pocketbase.io/docs/api-rules-and-filters/#lower-modifier) (_if you rely on a specific ORDER BY index you may have to adjust its column to `LOWER(yourColumn)`_).
